lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -18,6 +18,11 @@ admin_state_ethernet_noswitchport_shutdown:
     /N7K/:
       default_value: "shutdown"
 
+all_channel_groups:
```

Comment:
Why do you need `all_channel_groups` ? There can only be one channel group on an interface, right?